jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/cli2/JarCommandTest.java

```diff
@@ -142,6 +142,39 @@ public void testNoDependencyJar_packagedMode_toDocker()
     }
   }
 
+  @Test
+  public void testSpringbootLayeredJar_explodedMode_toDocker()
+      throws IOException, InterruptedException, URISyntaxException {
+    Path jarPath =
+        Paths.get(Resources.getResource("jarTest/springboot/springboot_layered.jar").toURI());
```

Comment:
How are these spring boot jars constructed? They don't seem like an actual jar built from a Spring Boot project. (Usually it runs a server listening on port 8080).

Reply:
Great question! To create the jars, I downloaded the demo project from https://start.spring.io/ and ran `./mvnw package`. Although, I just realized that I'm currently asserting on System.out instead of actually testing if the desired output shows up on localhost:8080 after having run `SpringApplication.run()` -- which is probably not a sign of a good test for a spring boot application. I'll correct this.

Reply:
Hmm... yeah. It looked weird that a Spring Boot app prints `Hello World` and quits. Do the tests actually pass?

We have similar Spring Boot integration tests, For example, take a look at

https://github.com/GoogleContainerTools/jib/blob/master/jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SpringBootProjectIntegrationTest.java